lib/ts/Arena.h

```diff
@@ -21,12 +21,13 @@
   limitations under the License.
  */
 
-#ifndef __ARENA_H__
-#define __ARENA_H__
+#pragma once
```

Comment:
I don't want to die-on-that-hill btw... but I am wondering what is good cleanup (i.e. a loop structure because the inside needs changing) and what is out-of-bounds busybody cleanup.